include/xgboost/data.h

```diff
@@ -42,34 +42,34 @@ class MetaInfo {
   static constexpr uint64_t kNumField = 9;
 
   /*! \brief number of rows in the data */
-  uint64_t num_row_{0};
+  uint64_t num_row_{0};  // NOLINT
```

Comment:
Why `NOLINT` here?

Reply:
Wrong naming.  Public member should not end with `_`.  But I don't want to refactor it in this PR as it's too massive.

Reply:
Got it.